src/Raven.Server/RavenServer.cs

```diff
@@ -1349,19 +1349,32 @@ private string GetWebUrl(string kestrelUrl)
             return Configuration.Core.ServerUrls[0];
         }
 
-        private CertificateHolder LoadCertificate()
+        private CertificateHolder LoadCertificateAtStartup()
         {
             try
             {
-                if (string.IsNullOrEmpty(Configuration.Security.CertificateExec) == false)
+                if (string.IsNullOrEmpty(Configuration.Security.CertificateLoadExec) == false &&
+                    (string.IsNullOrEmpty(Configuration.Security.CertificateRenewExec) || string.IsNullOrEmpty(Configuration.Security.CertificateChangeExec)))
                 {
-                    throw new InvalidOperationException($"Invalid certificate configuration. The configuration property '{RavenConfiguration.GetKey(x => x.Security.CertificateExec)}' has been deprecated since RavenDB 4.2, please use '{RavenConfiguration.GetKey(x => x.Security.CertificateLoadExec)}' along with '{RavenConfiguration.GetKey(x => x.Security.CertificateRenewExec)}' and '{RavenConfiguration.GetKey(x => x.Security.CertificateChangeExec)}'. For more information, refer to the online documentation at https://ravendb.net/l/4554RZ/4.2.");
```

Comment:
I'm 99% sure we don't need this restriction since we check that renew exec is not null before actually starting the renew process.